atlp-rwanda/champs-ec-fe
# #187854921 Change Dashboard Routes to Resolve Bug

#### What does this PR do?
Updates dashboard routes.
#### Description of Task to be completed?
Fixed the bug related to the dashboard route for product and profile sections, arranging them based on their relationships.
#### How should this be manually tested?
- Run npm run dev
- Check the deployed version
#### Any background context you want to provide?

#### What are the relevant pivotal tracker/Trello stories?

#### Screenshots (if appropriate)
![image](https://github.com/atlp-rwanda/champs-ec-fe/assets/131756888/d060b383-ccdb-494a-95ce-b09badf87c9f)

#### Questions:
